src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -458,16 +425,14 @@ public void NormalizePathBadUNC3()
         }
 
         [Fact]
-        [Trait("Category", "mono-osx-failing")]
-        [Trait("Category", "netcore-osx-failing")]
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
         public void NormalizePathGoodUNC()
         {
             Assert.Equal(@"\\localhost\share", FileUtilities.NormalizePath(@"\\localhost\share"));
         }
 
         [Fact]
-        [Trait("Category", "mono-osx-failing")]
-        [Trait("Category", "netcore-osx-failing")]
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
Did a few like this since they don't actually make sense xplat--it's msbuild features to work around Windows limitations.